tests/CMakeLists.txt

```diff
@@ -21,5 +21,9 @@ else()
   set(EXCLUDE_FROM_TESTS ${TESTDATA_TESTS})
 endif()
 
+if(NOT EXISTS ${FIVOXLFP_DIR}/lfpFunctor.h)
+  list(APPEND EXCLUDE_FROM_TESTS lfpValidation.cpp)
```

Comment:
"fivoxtest://" does not need any BlueConfig... I will try to fix that in the uirHandler